iBridges.py

```diff
@@ -38,15 +38,15 @@
 
 
 class IrodsLoginWindow(PyQt6.QtWidgets.QDialog,
-                       gui.ui_files.irodsLogin.Ui_irodsLogin,
-                       utils.context.ContextContainer):
+                       gui.ui_files.irodsLogin.Ui_irodsLogin):
     """Definition and initialization of the iRODS login window.
 
     """
     icommands = False
     this_application = ''
+    context = utils.context.Context()
 
-    def __init__(self):
+    def __init__(self, context):
```

Comment:
The passed `context` argument is not used.  Either choose to pass it and set it in the constructor as a class attribute or set it as an initial class attribute as above (L47).

Reply:
That's still an artefact, it should not be passed as a parameter.